server/app/config.py

```diff
@@ -36,7 +36,6 @@ class Config:
 
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     SQLALCHEMY_DATABASE_URI = DB_URL
-    SECRET_KEY = 'temp'
 
```

Comment:
```suggestion
SECRET_KEY = os.urandom(24)
```
Ref: https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY
https://stackoverflow.com/questions/34902378/where-do-i-get-a-secret-key-for-flask
Any good reason you removed it. Seems like its an important configuration. It would be left as blank if we dont set anything.

Reply:
Cool let's throw it back in. I removed it because having a plaintext secret key that's committed to a public repo is worse than nothing IMO.